stubs/openssl-python/METADATA.toml

```diff
@@ -1,4 +1,4 @@
 version = "0.1"
```

Comment:
Could you update this as well? `"20.0"` would be my best guess, but you will know better.

Reply:
Fixed.

Reply:
That doesn't work, unfortunately.
```
ERROR: Could not find a version that satisfies the requirement openssl-python==20.0.* (from versions: 0.1, 0.1.1)
ERROR: No matching distribution found for openssl-python==20.0.*```

It looks like there's confusion with package name, that tries to install https://github.com/the11/openssl-python, that is a different package completely, and the latest version is really 0.1.1.

Probably the folder name has to be changed to `pyopenssl`?

Reply:
Yes, I noticed this too, and opened #5647. :) But let's merge this first.

Reply:
Ok, reverted version back for now.

Reply:
Unfortunately, the last CI check fails, and I don't see a way to fix it without renaming the directory.

My suggestion is to merge as is.

Reply:
I'll try to disable stubtest temporarily. If we don't do that, unrelated PRs could fail. But I'll enable it again when we're moving it to the correct name.

Reply:
Thank you!